sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -93,6 +97,9 @@ internal class RadarApiClient(
 
         apiHelper.request(context, "GET", url, headers, null, false, object : RadarApiHelper.RadarApiCallback {
             override fun onComplete(status: RadarStatus, res: JSONObject?) {
+                if (status == RadarStatus.SUCCESS) {
+                    Radar.flushLogs()
```

Comment:
Is this circular logic here? 

`apiHelper.request` calls `flushLogs`
 ...which calls `apiClient.log`
 ... ...which calls `apiHelper.request`
 ... ... ...

Reply:
No - It's a different request. *this* call to flushLogs is inside a callback for hitting the "/config" endpoint. When the logs are flushed, its call to `apiHelper.request` uses a different callback, which does not flush the logs when complete.